### ISSUE TITLE
Problem with alert "input:" in io.read() call

### ISSUE THREAD

**Issue by KataPi** [daurnimator/lua.vm.js#58]
Hi  daurnimator,
I'm not a programmer. I know js on a basic level and I'm learning Lua.
I love your work and it helps me to develop scripts locally.

Local and online (http://daurnimator.github.io/lua.vm.js/repl.html link from GitHub) when I call io.read()  lua.vm.js returns me an allert window ("input: ") and then another one and then another one ...
`mee=io.read()
print(mee)`

In google I found http://kripken.github.io/mloc_emscripten_talk/lua/repl.html#
This works well. In io.read () call it give me back only one alert. It works!
It's a bug? It's my browser??

Html from second link (http://kripken.github.io/mloc_emscripten_talk/lua/repl.html#) page not work in local with lua.vm.js from your GitHub.
What can I do?
Can you give me vm from http://kripken.github.io/mloc_emscripten_talk/lua/repl.html# ?
or can you fix vm in GitHub?

Tks
Kata

**Comment by daurnimator** [comment]
heh. I didn't even know stdin was hooked up to [`prompt`](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt).

Looks like this is an emscripten bug. I guess you'll have to file it over there? ping @kripken

**Comment by KataPi** [comment]
Hi daurnimator,
thanks for reply.

I do not think is an emscripten bug. In his page it works.
When I built/installed emscripten from your GitHub with make it gave me a lot of errors but my local page works as your web page, so I did not worry...

Your ping calls him on this issue?
Waiting @kripken ...

Tks
Kata

**Comment by kripken** [comment]
By default emscripten hooks up `stdin` to `window.prompt`, yes. It sounds like something changed in that handling, as an old version worked but not the new one? The best thing would be to bisect in both repos, I guess, to find out what changed.

**Comment by daurnimator** [comment]
@kripken Wouldn't it be bisecting emscripten that's helpful here?

As far as I can tell, it's something weird happening inside of fread/libc/syscall emulation, rather than in lua.vm.js.

**Comment by kripken** [comment]
Yes, but if you upgraded lua here, might need to bisect that too (might use different syscalls).
